src/hip/handlehip.cpp

```diff
@@ -57,24 +56,17 @@
 #include <mutex>
 #include <shared_mutex>
 
-#define MIOPEN_WORKAROUND_ROCM_COMPILER_SUPPORT_ISSUE_30 \
-    (MIOPEN_USE_COMGR && BUILD_SHARED_LIBS && (HIP_PACKAGE_VERSION_FLAT < 4003000000ULL))
-
+/// GFX906 and GFX103X are deprecated since 5.7 RC.
```

Comment:
Why is GFX103X deprecated?

Reply:
@V6ser Not in MIOpen, in HIP runtime. I have no idea why. It still works, but I see some quirks like this one.

Reply:
@atamazov I would recommend remove or revise this line. It's a bug and issue on hip runtime nevertheless.

Reply:
@junliume I agree.